### PR TITLE
write tree structure to file

### PR DESCRIPTION
**Pull request by lorenzwalthert**
Writes tree structures in the respective test folder as text files (without extension) so they can be opened in a text editor or Rstudio conveniently.
@krlmlr Should we commit these files as well? They are many but small, so I guess it makes sense since you want to have a look at those without having to run all test first...

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -13,7 +13,8 @@ Imports:
     tidyr,
     purrr,
     magrittr,
-    pkgload
+    pkgload,
+    readr
```
Comment (lorenzwalthert) [inline DESCRIPTION]: alphabetically you mean, right?

Comment (krlmlr) [inline DESCRIPTION]: Yes, please

**Comment by lorenzwalthert** [comment]
Ok. Should I commit the tree output files also?

**Comment by krlmlr** [comment]
Yes, please commit the tree files too, so we can see what's happening at a glance.

File: tests/testthat/indention_round_brackets/arithmetic_no_start-in_tree [hunk 2]
```diff
@@ -0,0 +1,19 @@
+levelName
+xxx
```
Comment (krlmlr) [inline tests/testthat/indention_round_brackets/arithmetic_no_start-in_tree]: Can we make this a bit nicer?

Comment (lorenzwalthert) [inline tests/testthat/indention_round_brackets/arithmetic_no_start-in_tree]: Yes, I made it look a bit better.
```
ROOT (token: short_text [newlines/spaces])
  °--expr:  [0/0]                          
      ¦--'(': ( [0/0]   
```

**Pull request merged**